### DISCUSSION TITLE
plan_id when the reporting_entity_type is "third-party administrator",

### DISCUSSION THREAD

**Discussion by nmayle** [CMSgov/price-transparency-guide#529] [Q&A]
When the reporting entity is a third party administrator, I assume that the plan_id when it is an EIN should be the EIN of the sponsor and not of the administrator.  The language in the documentation is a bit vague "the Employer Identification Number (EIN)for each plan or coverage offered by a plan or issuer."  

I am seeing TOC files where the EIN stored as the plan_id is for the administrator.  

Which is right?

**Comment by CKreklow** [comment]
I agree with your interpretation, we are reporting the EIN of the plan for self-funded, not the EIN of the TPA.
